drivers/scale_service.go

```diff
@@ -44,6 +44,10 @@ func (s *ScaleServiceDriver) ValidatePayload(conf interface{}, apiClient client.
 		return http.StatusBadRequest, fmt.Errorf("Maximum scale not provided/invalid")
 	}
 
+	if config.Min > config.Max {
```

Comment:
@cjellick yes actually. I initially thought that there doesn't have to be an error thrown in that case. But I'll add that now to the condition